test/suites/dev/moonbase/test-balance/test-balance-existential.ts

```diff
@@ -1,9 +1,11 @@
 import "@moonbeam-network/api-augment";
 import { expect, describeSuite, beforeEach, TransactionTypes } from "@moonwall/cli";
-import { ALITH_ADDRESS, baltathar, GLMR, MIN_GAS_PRICE } from "@moonwall/util";
+import { ALITH_ADDRESS, baltathar, GLMR } from "@moonwall/util";
 import { createRawTransfer } from "@moonwall/util";
 import { Wallet } from "ethers";
 
+const MIN_GAS_PRICE = 2500000000n;
```

Comment:
This should be in `test/helpers/constants.ts`

Reply:
I think it should be fixed in Moonwall no ?
let's move it to constant for now yes

Reply:
I think it is better in constants, you are right